src/templates/layouts/vetCenter/index.test.tsx

```diff
@@ -0,0 +1,198 @@
+import { render, screen } from '@testing-library/react'
+import { VetCenter } from './index'
+
+const testData = {
```

Comment:
i'd try and pull this in using the mock data + formatter, similar to how `eventListing.test.tsx` is doing it. i realize it is the only layout currently doing so, the others should be refactored (not your problem in this PR)